.husky/commit-msg

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
```

Comment:
Why are we editing this file. It should be @Homoakin619 or someone else in DevOps responsibility